zulipterminal/helper.py

```diff
@@ -275,3 +276,17 @@ def classify_unread_counts(unread_msg_counts: Dict[str, Any])\
         unread_counts['all_msg'] += count
 
     return unread_counts
+
+
+def match_user(user: Any, text: str) -> bool:
+    """
+    Matches if the user full name, last name or email matches
+    with user or not.
+    """
+    keywords = user.caption.split()
```

Comment:
The search is case sensitive.  It might be worth making this case insensitive. 
![image](https://user-images.githubusercontent.com/315678/40440950-2124ea54-5edd-11e8-8017-5397f2ca5f5a.png)

Reply:
We do everything case-insensitive in Zulip with names/stream names/topics/etc., so that's probably the right way to go here as well.  Might be worth doing a quick codebase audit for similar issues.